ONSdigital/rm-tools
# Add collection exercises for next year

# Motivation and Context
We are having problems with Bricks collection exercises in our acceptance tests, because we were using a collection exercise which is now past its go-live date. We will be having a number of problems during the rest of this month and next, because we have 'kicked the can down the road' by setting dates in the not-too-distant future, which are all triggering now.

**There are currently no collection exercises in 2019 at all**

This is a stop-gap PR to create a load of collection exercises we can use until we refactor all our automated tests to not use hard-coded collection exercises which occur on specific dates.

# What has changed
Added a complete set of collection exercises for 2019, where exercises existed for 2018 and were likely being used by the automated acceptance tests.

# How to test?
Hack acceptance tests scripts to pull rm-tools from this branch instead of master. Then run `make setup` and check that all the 2019 collection exercises get created.

# Links
Trello: https://trello.com/c/DOsrckW2/585-bug-fix-the-ci-pipeline-failure-due-to-collection-exercise-go-live-date
